website/pages/blog/announcing-premium-plugins.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Moving Preview Plugins to Premium
+tag: announcement
+date: 2023/11/06
+description: Preview Plugins Are Moving to Premium Preview
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Development and maintenance of our open source plugins requires a substantial effort on our side. We have decided that going forward, new versions of some of our preview plugins will be available as closed-source. While older versions will have the source available in our [git repository](https://github.com/cloudquery/cloudquery), premium plugins will be available only via our [Hub.](https://hub.cloudquery.io/)
```

Comment:
```suggestion
Development and maintenance of our open source plugins requires a substantial effort on our side. We have decided that going forward, new versions of some of our preview plugins will be available as closed-source. While older versions will have the source available in our [git repository](https://github.com/cloudquery/cloudquery), premium plugins will be available only via our [Hub](https://hub.cloudquery.io/).
```

Reply:
the full stop needs to be in the link otherwise it gets rendered on a new line :/

Reply:
Try with space? `[Hub](https://hub.cloudquery.io/) .`?

Reply:
```md
[Hub](https://hub.cloudquery.io/) .
```

[Hub](https://hub.cloudquery.io/) .

```
[Hub](https://hub.cloudquery.io/).
```

[Hub](https://hub.cloudquery.io/).

& some footer.